Lib/unittest/test/testmock/testmock.py

```diff
@@ -925,6 +925,50 @@ def test_mock_calls(self):
                              call().__int__().call_list())
 
 
+    def test_child_mock_call_equal(self):
+        m = Mock()
+        result = m()
+        result.wibble()
+        # parent looks like this:
+        self.assertEqual(m.mock_calls, [call(), call().wibble()])
+        # but child should look like this:
+        self.assertEqual(result.mock_calls, [call.wibble()])
+
+
+    def test_mock_call_not_equal_leaf(self):
+        m = Mock()
+        m.foo().bar()
+        self.assertNotEqual(m.mock_calls[1], call.foo().baz())
```

Comment:
🤣 I've been scratching might head for 3 minutes until I noticed the `baz` rather than `bar`